core/src/scene/sceneLoader.cpp

```diff
@@ -156,14 +156,14 @@ void SceneLoader::applyGlobals(Node root, Scene& scene) {
     }
 }
 
-bool SceneLoader::applyConfig(const std::shared_ptr<Scene>& _scene) {
+bool SceneLoader::applyConfig(const std::shared_ptr<Platform>& _platform, const std::shared_ptr<Scene>& _scene) {
 
     Node& config = _scene->config();
 
     // Instantiate built-in styles
     _scene->styles().emplace_back(new PolygonStyle("polygons"));
     _scene->styles().emplace_back(new PolylineStyle("lines"));
-    _scene->styles().emplace_back(new DebugTextStyle("debugtext", true));
+    _scene->styles().emplace_back(new DebugTextStyle("debugtext", std::make_shared<FontContext>(_platform), true));
```

Comment:
The behavior stays the same here, see https://github.com/tangrams/tangram-es/pull/1263/files/7ebd47ea2232dab4febaf26c21363126738046b3#diff-7822ef08be0ac1769ea05762681bbf15L22.

The debug text style would have no overhead synchronizing on regular text rendering; I also didn't want to introduce changes in this behavior on this PR.